src/main/java/com/xiaomi/infra/pegasus/client/PConfigUtil.java

```diff
@@ -77,13 +74,25 @@ public static Properties loadConfiguration(String configPath) throws PException
                 + "but actual configPath is "
                 + configPath);
       }
+      if (stream == null) {
+        throw new PException("config resource not found: " + configPath);
+      }
+      config.load(stream);
```

Comment:
```suggestion
      config.load(stream);
      stream.close();
```

Close here. So you don't have to add another try-catch after `finally`.

Reply:
If `config.load` throws an exception, resources cannot be released normally.😂

Reply:
Sure. But the stream will not be closed before `config.load` works are done.

```suggestion
      config.load(stream); // if any exception, it will be thrown out, won't steps down to `stream.close()`
      stream.close();
```

Reply:
Although the `config.load` works is not completed, I think the resource also  needs to be released.